src/models/disassemblyoutput.cpp

```diff
@@ -271,10 +271,12 @@ DisassemblyOutput DisassemblyOutput::disassemble(const QString& objdump, const Q
                                   toHex(symbol.relAddr + symbol.size)};
 
     // only available for objdump 2.34+
-    if (canVisualizeJumps(processPath))
+    auto canShowBranches = canVisualizeJumps(processPath);
```

Comment:
This may be a good time to recheck why this function is called twice - when you use the disassembly function and it the jumps are not supported, then the message about missing support is seen twice.

Reply:
somehow `disassemble` is called twice but I haven't investigated further

Reply:
I _guess_ fixing that will also speed up handling of huge functions. Thanks for having a look at this later.